stroeer/page/section/v1/section_page.proto

```diff
@@ -59,51 +60,49 @@ message StageConfiguration {
   repeated stroeer.core.v1.Reference references = 2;
 }
 
-// A single generic item of a stage.
+// A generic model to represent what can be part of a stage.
 message StageItem {
```

Comment:
this is virtually the same as `message TeaserStageItem` only difference is an allowed/forbiden field implemented as `oneof`. And again, `Commercial` and `ArticleTeaser` only differ in one field, but is a different message and is using a different mechanism (separate message).
This feels a bit inconsistent to me.

Reply:
Yep you're right. What would your proposal to remove the recursive stages after level 2?

Reply:
I do not like this move, instead we can somehow guarantee this by contract and get rid of this duplication?